src/tests/ftest/util/ior_test_base.py

```diff
@@ -173,8 +174,14 @@ def update_ior_cmd_with_pool(self):
         # Create a pool if one does not already exist
         if self.pool is None:
             self.create_pool()
-        # Update IOR params with the pool
-        self.ior_cmd.set_daos_params(self.server_group, self.pool)
+        # Always create a container
+        # Don't pass uuid and pool handle to IOR.
+        # It will not enable checksum feature
+        self.pool.connect()
+        self.create_cont()
```

Comment:
Are we always creating a container of type POSIX for each IOR run here?

Reply:
As discussed, not necessary.